.github/workflows/ci.yml

```diff
@@ -22,5 +22,11 @@ jobs:
       uses: actions/setup-node@v2
       with:
         node-version: ${{ matrix.node-version }}
-        cache: ${{ !env.ACT && 'npm' || '' }}
-    - run: npm test
+        cache: ${{ !env.ACT && 'yarn' || '' }}
+    - name: Setup Yarn
+      run: npm install -g yarn
+    - name: Run Tests
+      run: yarn test
+    - name: Uninstall Yarn
+      if: always()
+      run: npm uninstall -g yarn
```

Comment:
What's the reason for uninstall? This makes the CI run longer, while the environment is discarded at the end of the build anyways.

Reply:
That comment makes sense, however we're not using a self-hosted runner. We're using github-provided runner instead.

Reply:
We're using hosted runners, which use https://github.com/actions/virtual-environments
As far as I understand, in practical terms it means env is a new docker container from the virtual env image, so state doesn't transfer.

Reply:
Okay, i will remove it.